python/cudf_polars/cudf_polars/testing/asserts.py

```diff
@@ -128,11 +128,14 @@ def assert_gpu_result_equal(
     else:
         tol_kwargs = {"rel_tol": rtol, "abs_tol": atol}
 
+    # the type checker errors with:
+    # Argument 4 to "assert_frame_equal" has incompatible type "**dict[str, float]"; expected "bool"  [arg-type]
+    # which seems to be a bug in the type checker / type annotations.
```

Comment:
Can you say more? If its a bug in the annotations, can we fix them?

Reply:
https://mypy-play.net/?mypy=latest&python=3.12&gist=aa15d6a936588cab83ec613ae030b410 has an example. Looks to be a bug in mypy, but I'm not sure and wasn't sure what to search for.

Reply:
Should we link the mypy bug report instead?

Reply:
I'm not sure if there is one, and I wasn't sure what to search for to find one. Not worth tracking this down IMO, since this is only used in testing.

Reply:
Thanks, makes sense